frontend/appflowy_flutter/lib/workspace/presentation/home/menu/menu.dart

```diff
@@ -104,6 +108,56 @@ class HomeMenu extends StatelessWidget {
     );
   }
 
+  Widget _renderFavorites(BuildContext context) {
+    return BlocProvider(
+      create: (context) =>
+          getIt<FavoriteBloc>()..add(const FavoriteEvent.initial()),
+      child: BlocConsumer<FavoriteBloc, FavoriteState>(
+        listener: (context, state) {
+          Log.warn("Favorite state changed $state");
+        },
+        builder: (context, state) {
+          return state.objects.isNotEmpty
+              ? ExpandableTheme(
+                  data: ExpandableThemeData(
+                    useInkWell: true,
+                    animationDuration: Durations.medium,
+                  ),
+                  child: ExpandablePanel(
+                    theme: const ExpandableThemeData(
+                      headerAlignment: ExpandablePanelHeaderAlignment.center,
+                      tapBodyToExpand: false,
+                      tapBodyToCollapse: false,
+                      tapHeaderToExpand: false,
+                      iconPadding: EdgeInsets.zero,
+                      hasIcon: false,
+                    ),
+                    header: FavoriteHeader(),
+                    expanded: ScrollConfiguration(
+                      behavior:
+                          const ScrollBehavior().copyWith(scrollbars: false),
+                      child: Column(
+                        children: state.objects
+                            .map(
+                              (e) => ViewSectionItem(
+                                key: ValueKey(e.id),
+                                isSelected: false,
+                                onSelected: (item) {},
+                                view: e,
+                              ),
+                            )
+                            .toList(),
+                      ),
+                    ),
+                    collapsed: const SizedBox(),
+                  ),
+                )
+              : const Offstage();
```

Comment:
Why `Offstage` here rather than `const SizedBox.shrink()`, Offstage is worse for performance.